overlord/devicestate/export_test.go

```diff
@@ -413,3 +415,37 @@ func MockSecbootRemoveRecoveryKeys(f func(mountPointToRecoverKeyFile map[string]
 	secbootRemoveRecoveryKeys = f
 	return restore
 }
+
+func MockOsutilAddUser(addUser func(name string, opts *osutil.AddUserOptions) error) (restore func()) {
+	oldAddUser := osutilAddUser
+	// internal.SetOsutilAddUser(addUser)
+	osutilAddUser = addUser
+	return func() {
+		osutilAddUser = oldAddUser
+		// internal.SetOsutilAddUser((func(name string, opts *osutil.AddUserOptions) error)(oldAddUser))
+	}
```

Comment:
You can use testutil.Backup() here too.

I'd remove the commented-out code.

Reply:
Fixed